pypeit/spectrographs/keck_nires.py

```diff
@@ -171,6 +154,7 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
         """
         Check for frames of the provided type.
         """
+        # TODO: Arcs, tilts, darks?
```

Comment:
This is not my TODO, and I've never altered init_meta.

Reply:
No, it was mine.  The point of the TODO was to note that this spectrograph doesn't type these frames.  If that doesn't matter, we can move on, but we should note this in the docstring.  I'd like to just leave it as a TODO for now.